dev/TabView/TestUI/TabViewPage.xaml

```diff
@@ -103,66 +103,63 @@
                         </Grid>
                     </controls:TabView.TabStripFooter>
 
-                    <controls:TabView.TabItems>
```

Comment:
Is the old syntax also still valid? 

In ListView, both of these work: 
```
<ListView>
    <ListView.Items>
        <ListViewItem Content="Item" />
        <ListViewItem Content="Item" />
        <ListViewItem Content="Item" />
    </ListView.Items>
</ListView>
```
and
```
<ListView>
    <ListViewItem Content="Item" />
    <ListViewItem Content="Item" />
    <ListViewItem Content="Item" />
</ListView>
```

Reply:
Yes. Both should work. The attribute is a hint to the parser to allow skipping the tag in markup. 
https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Markup.ContentPropertyAttribute

Reply:
Sounds good. If so, should this test file be reverted since it's not strictly necessary? And/or should there be another test that validates that both syntaxes are valid?

Reply:
yes. strictly not required but it is good to know if we ever remove that attribute by accident. and yes there is another page where we use the explicit syntax so we are covered.